File: Java/Primes.java
import java.util.*;

public class Primes {
  public static void main(String[] args) {
    System.out.println(primeFinder(13));
  }

  public static ArrayList primeFinder(int n) {
    ArrayList<Integer> primes = new ArrayList<Integer>();
    primes.add(2);

    for(int i = 2; i <= n; i++) {
      boolean prime = true;

      for(int j = 0; j < primes.size(); i++) {
        if(i % primes.get(j) == 0) {
          prime = false;
        }
      }

      if(prime) {
        primes.add(i);
      }
    }

    return primes;
  }
}

File: Python/Primes.py
def prime_finder(n):
  primes = [2]

  for i in range(2, n + 1):
    prime = True
    for j in range(0, len(primes)):
      if i % primes[j] == 0:
        prime = False

    if prime:
      primes.append(i)
  
  return primes
 
print(prime_finder(11))
